.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,60 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+    inputs:
+      subsystem:
+        description: Subsystem to run the update for
+        required: false
+        default: url
```

Comment:
This doesn't give way to have it run for multiple subsystems at a later point, please see the daily wpt fyi runner.

Reply:
can ı use it `default: '["current", "lts/*", "lts/-1"]'`

Reply:
This is tied to how you construct the matrix later. As is it won't work.

Reply:
I think I need to make it compatible with the url